packages/start-plugin-core/src/plugin.ts

```diff
@@ -125,7 +138,7 @@ export function TanStackStartVitePluginCore(
           define: {
             ...injectDefineEnv('TSS_PUBLIC_BASE', startConfig.public.base),
             ...injectDefineEnv('TSS_CLIENT_BASE', startConfig.client.base),
-            ...injectDefineEnv('TSS_CLIENT_ENTRY', startConfig.clientEntryPath),
+            ...injectDefineEnv('TSS_CLIENT_ENTRY', `/@fs${getClientEntryPath(startConfig)}`), // This is consumed by the router-manifest, where the entry point is imported after the dev refresh runtime is resolved
```

Comment:
An example of the value printed here is `/@fs/Users/seanc/code/tanstack/router/e2e/react-start/src/client.tsx`).

The `/@fs` prefix is something Vite lets you use to access the file-system. This is erased by Vite at build time.